product_brand/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# © 2009 NetAndCo (<http://www.netandco.net>).
+# © 2011 Akretion Benoît Guillot <benoit.guillot@akretion.com>
+# © 2014 prisnet.ch Seraphine Lantible <s.lantible@gmail.com>
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# © 2018 Daniel Campos <danielcampos@avanzosc.es>
+# Copyright 2018 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    'name': 'Product Brand Manager',
+    'version': '12.0.1.1.0',
+    'category': 'Product',
+    'summary': "Product Brand Manager",
+    'author': 'NetAndCo, Akretion, Prisnet Telecommunications SA, '
+              'MONK Software, SerpentCS Pvt. Ltd., Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/product-attribute',
+    'license': 'AGPL-3',
+    'depends': [
+        'sale',
+        ],
+    'data': [
```

Comment:
@ThomasBinsfeld Improve Sequence as per [this](https://github.com/odoo/odoo/blob/11.0/addons/hr_recruitment/__manifest__.py#L20)

Reply:
@nikul-serpentcs I'm not sure to understand.

Reply:
@ThomasBinsfeld  IMO improve file position like [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L27).

always first `security` and after `views` and `reports`.

Reply:
Done